sigstore-java/src/main/java/dev/sigstore/KeylessVerifier.java

```diff
@@ -208,4 +206,18 @@ public void verify(byte[] artifactDigest, Bundle bundle, VerificationOptions opt
           "Signature could not be processed: " + ex.getMessage(), ex);
     }
   }
+
+  @VisibleForTesting
+  void checkCertificateMatchers(X509Certificate cert, List<CertificateMatcher> matchers)
+      throws KeylessVerificationException {
+    try {
+      if (matchers.size() > 0 && matchers.stream().noneMatch(matcher -> matcher.test(cert))) {
+        throw new KeylessVerificationException(
+            "No provided certificate identities matched values in certificate");
```

Comment:
would it be helpful here to include matcher spec in the exception message?

Reply:
yeah probs

Reply:
done